bigquery.js

```diff
@@ -0,0 +1,92 @@
+var BigQueryClient = require('google-cloud/node_modules/@google-cloud/bigquery');
+
+const projectData = require('./auth.json');
+
+const config = {
+  projectId: projectData['project_id'],
+  keyFilename: './auth.json'
+};
+
+const ds = 'atomic';
+
+const PARTITION_FIELDS = process.env['PARTITION_FIELDS'].split(',');
+
+var BigQuery = function() {
+  this.conn = BigQueryClient(config);
+};
+
+BigQuery.prototype.insertInto = function(tableName, rows, options) {
+  //Here I should get the table based on name
+  // and insert in the right partition whenever the table is partitioned
+  rows.forEach((row) => {
+    // First, we get the table object with all available information
+    this.table(tableName, (err, table) => {
+      if (err) {
+        console.log(`Sorry! We've got an error while fetching the table ${tableName}.\nERROR: ${err}`);
+        return;
+      }
+      var isPartitioned = BigQuery.isPartitioned(table);
+      var tablePartition = isPartitioned ? BigQuery.partitionName(row, table) : undefined;
+      // Insert if table is not partitioned or partition column is not defined
+      if (!isPartitioned || tablePartition === undefined) {
+        table.insert(row, (options || {}), function(err, insertErrors, apiResponse) {
```

Comment:
What do you thing about move the `(option || {})` to method parameters.

```javascript
BigQuery.prototype.insertInto = function(tableName, rows, options = {}) {...
```

Reply:
It doesn't work. I've tested.